ProjectStructureAndRoles.md

```diff
@@ -48,7 +101,7 @@ In the Project the “Fork and pull model” is used. Changes and contributions
 
 2 Either the issue creator or one of the Sub Project Maintainers then shall assign the issue to one of the Contributors of the Sub Project.
 
-3 This Contributor shall create a fork (if necessary), shall create a new branch, and shall copy the branch link into the issue. For the naming of the branch a clear and concrete description shall be used.
+3 This Contributor shall create a fork (if not yet done before), shall create a new branch within the fork, and shall copy the branch link into the issue. For the naming of the branch a clear and concrete description shall be used.
```

Comment:
Would copying the link in the issue be necessary? If we use the PR template and the #fixes tag, then this copying step is not needed. Probably not relevant for this PR but added this as this statement was part of the change.

Reply:
You are right, by using the "fixes" logic of GitHub, the PR will get automatically copied into the issue. But it is also possible to Lin the branch already earlier (there is a field within a GitHub issue for that, which is e.g. filled if the branch will be created out of the issue). Making this optional could be the scope of a separate PR.